package.json

```diff
@@ -64,7 +64,7 @@
     "d3": "3.5.5",
     "datatables.net": "1.10.10",
     "datatables.net-responsive": "2.0.1",
-    "fec-style": "1.4.1",
+    "fec-style": "git://github.com/jmcarp/fec-style.git#consolidate-typeahead",
```

Comment:
Just noting that this will have to change once the updates are applied to the `fec-style` repo.